<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="CentrosomeImageQLabel" name="movieImgLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTreeView" name="experimentsTreeView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>511</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
  </widget>
  <widget class="QListView" name="nucleiListView">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>81</width>
     <height>431</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="nucleiTitleLbl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>56</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuclei</string>
   </property>
  </widget>
  <widget class="MplWidget" name="mplDistance" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>941</width>
     <height>321</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLineEdit" name="gaborLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="gaborThresholdLbl">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gabor Filter threshold</string>
   </property>
  </widget>
  <widget class="QPushButton" name="renderBoxplotButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Render Boxplot</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>gui_mplwidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CentrosomeImageQLabel</class>
   <extends>QLabel</extends>
   <header>gui_centrosome_image_qlabel</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
